learning_management_services/demo/sale_order_confirm.xml

```diff
@@ -0,0 +1,287 @@
+
+    <record id="sale_advance_payment_inv_13" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_16')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_13')]"/>
+
+    <record id="sale_advance_payment_inv_14" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_17')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_14')]"/>
+
+    <record id="sale_advance_payment_inv_15" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_18')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_15')]"/>
+
+    <record id="sale_advance_payment_inv_16" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_21')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_16')]"/>
+
+    <record id="sale_advance_payment_inv_17" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_22')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_17')]"/>
+
+    <record id="sale_advance_payment_inv_18" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_34')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_18')]"/>
+
+    <record id="sale_advance_payment_inv_19" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_35')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_19')]"/>
+
+    <record id="sale_advance_payment_inv_20" model="sale.advance.payment.inv">
+        <field name="advance_payment_method" >delivered</field>
+        <field name="sale_order_ids" eval="[Command.set([ref('sale_order_36')])]"/>
+    </record>
+    <function model="sale.advance.payment.inv" name="create_invoices"
+        eval="[ref('sale_advance_payment_inv_20')]"/>
```

Comment:
```suggestion
```

no need to use this as we can directly create invoice by `_create_invoices` function